Open Judge System/Workers/OJS.Workers.LocalWorker/SubmissionJob.cs

```diff
@@ -228,7 +228,7 @@ private void ProcessSubmission(Submission submission)
             }
             catch (Exception exception)
             {
-                this.logger.ErrorFormat("executionStrategy.Execute on submission №{0} has thrown an exception: {1}", submission.Id, exception);
+                this.logger.Error($"executionStrategy.Execute on submission №{submission.Id} has thrown an exception:", exception);
```

Comment:
What gets logged here with multiple nested inner exceptions like Db.Update/OptimisticConcurrencyException? Do we get the full stack trace?

Reply:
Yes, we get all inner exceptions with all of their inner exceptions